recipes/wg21-linear_algebra/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class LAConan(ConanFile):
+    name = "wg21-linear_algebra"
```

Comment:
The name looks a little weird mixing `-` and `_` but it's valid

Reply:
Yes, agreed, but it matches the repo name.  Should this proposal be accepted in the C++ standard then there the wg21 prefix is there to distinguish between the proposal and a linear_algrebra accepted into the standard (which would presumably be std-linear_algebra)